README.md

```diff
@@ -125,6 +125,8 @@ vars:
     using_domain_names:         False         #Disable if you are not using domain names
     using_user_tags:            False         #Disable if you are not using user tags
     using_ticket_form_history:  False         #Disable if you are not using ticket form history
+    using_brands:               False         #Disable if you are not using brands
+    using_organizations:        False         #Disable if you are not using organizations
```

Comment:
Do we want to also callout that once `using_organizations` is disabled, it not only disables organization but also organization tags?

Reply:
We definitely should! Added.